### PR TITLE
ui: Settings screen according to figma

### PR DESCRIPTION
**Pull request by theborakompanioni**
Adapt `Settings` screen according to Figma.

Differences:
- "Log out" is in section "Wallet" instead of "Settings"

Question:
- Should the "Create new wallet" button be removed? Personally, leaning towards "Yes". (related https://github.com/joinmarket-webui/joinmarket-webui/issues/228)

### CODE REVIEW

**Comment by editwentyone** [comment]
the Logout is for the whole app, not on wallet level. it should go over Wallet category please :)

**Comment by theborakompanioni** [comment]
> maybe even "Log out from Jam"

Hmm.. I am not sure I know what this means.

Currently "logging out" is just forgetting the session key. What should happen, when someone "logs out"?

**Comment by editwentyone** [comment]
its should bring you to "login page" https://www.figma.com/file/kfejZJFlwBywvLEnPEmJo1/?node-id=3994%3A73345

but I think we do not have that yet

**Comment by dnlggr** [comment]
> the Logout is for the whole app, not on wallet level.

I think these are equivalent things at this time (as @theborakompanioni mentioned) right?

File: src/constants/routes.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export const routes = {
   home: '/',
```
[the remark this answers is not in the document]
Comment (dnlggr) [reply src/constants/routes.ts:2]: No but I mean `routes.home` and `routes.walletList` are a duplicate, right?

**Comment by editwentyone** [comment]
I think that "log out" from Wallet, because its now underneath is misleading. 

logout should be only used when the user logged in somewhere, if that's not the case, we need to rethink what's actually happening.

I would move logout back up again and hide it as long as we don't have login implemented. why would a user logout aka "lock" the wallet and not just close the window?

**Comment by theborakompanioni** [comment]
> I would move logout back up again and hide it as long as we don't have login implemented. 

Removed it for now.

> why would a user logout aka "lock" the wallet and not just close the window? 

You are right, the (now removed) "Log out" action, by forgetting the auth token, would have been the same as closing the window.
However, I can relate to some _uneasiness_ when closing a tab with an "unlocked" wallet. A dedicated button would "feel" a little better, albeit there is no difference in what actually happens.

> what happens when we just **close** the window?

The auth token is removed (as `sessionStorage` cleared when the page session ends).

> 
> ~
> 
> so we have:
> 
>     * **create wallet**: creating new wallets, not possible while another wallet is still used and unlocked
> 
>     * **unlock wallet**: wallet is _inactive_, **needs password** to be active
> 
>     * **open wallet**: entering, using, wallet is _active_
> 
>     * **lock wallet**: deactivate wallet, wallet is now _inactive_
> 
>     * **Logout** wallet (misunderstanding): wallet stays active in the background, **needs password** to be opened/ entered again

Maybe something like "Leave wallet", "Exit wallet" or "Close wallet" (although the term "Closing" might be confused by "Locking")?

In any case, I have just removed it for now and will support anything that you think makes most sense from an UI/UX perspective.
Do you think these changes can be merged?

**Comment by dnlggr** [comment]
Removing seems like n acceptable solution for now. 👍  All the nuances are way to complicated, we really need to figure out a good concept for this.

---

IMG_F1CC67BC9C25-1

A little padding for the buttons on mobile would be great.

**Comment by theborakompanioni** [comment]
> A little padding for the buttons on mobile would be great.

Yeah, did not change that, as I supposed it was intentional. Any improvements are very welcome! Feel free to just push to this branch!

**Comment by dnlggr** [comment]
> Yeah, did not change that

Good point! :D Will push an update.

**Review by editwentyone — APPROVED** [review]
hidden is ok for me, I sketched already something for the next call, and yeah: its confusing :P

**Comment by 0xSaksham** [comment]
@theborakompanioni, I was taking inspiration for ConfirmModal use case and found out that this feature doesn't work here. If you want I can work on it?

**Comment by theborakompanioni** [comment]
> @theborakompanioni, I was taking inspiration for `ConfirmModal` use case and found out that this feature doesn't work here. If you want I can work on it?

I am not quite sure what you are referring to.. can you elaborate?

**Comment by 0xSaksham** [comment]
Confirm modal wants you to confirm that you really want to do this.

But there's no modal opening up here.

**Comment by 0xSaksham** [comment]
@theborakompanioni let's discuss this in tomorrow's meeting.

If there's no meeting, we'll discuss this in one-on-one call.

**Pull request merged**